src/utils.ts

```diff
@@ -203,3 +203,12 @@ const urlsWebApp: any = [
 export function getCurrentPageForUrl(currentUrl: string): any {
   return urlsWebApp.find((item: any) => item.url === currentUrl);
 }
+
+export function findUrlInRetryList(currentUrl: any): any {
```

Comment:
`(currentUrl: any)` why not `string`?

Reply:
can be null

Reply:
In that case:

```suggestion
export function  isUrlInRetryList(currentUrl: string | null): bool {
```

Reply:
but if find the url return the element dont a boolean

Reply:
> but if find the url return the element dont a boolean

I know that you can fix that :P

As I understand, you do not want to find a URL, you want to verify if it is in the list, for that reason I purpose to change the name of the function and the result type.

Reply:
fixed